### PR TITLE
Anchore Grype Mapper

### PR DESCRIPTION
**Pull request by andytang99**

### CODE REVIEW

File: libs/hdf-converters/sample_jsons/trufflehog_mapper/sample_input_report/trufflehog_docker_example.json
Comment (charleshu-8) [inline libs/hdf-converters/sample_jsons/trufflehog_mapper/sample_input_report/trufflehog_docker_example.json:1]: Revert these changes to Trufflehog. Keep the scope of this PR locked to Grype.

File: libs/hdf-converters/sample_jsons/trufflehog_mapper/trufflehog_to_hdf_example.json
Comment (charleshu-8) [inline libs/hdf-converters/sample_jsons/trufflehog_mapper/trufflehog_to_hdf_example.json:1]: Revert these changes to Trufflehog. Keep the scope of this PR locked to Grype.

File: libs/hdf-converters/src/utils/fingerprinting.ts
Comment (charleshu-8) [inline libs/hdf-converters/src/utils/fingerprinting.ts:1]: Friendly reminder to add fingerprints.

**Review by charleshu-8 — COMMENTED** [review]
Quick comments fr/ something I noticed fr/ our last meeting.

**Review by Amndeep7 — APPROVED** [review]
